android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -276,6 +276,11 @@ private fun Answer(type: QuestionDisplayType, onAnswersChange: (List<AnswerInput
             placeholder = type.placeholder,
             modifier = Modifier.height(170.dp)
         )
+        is QuestionDisplayType.Textfield -> FormAnswer(
+            placeholders = type.placeholders,
+            onTextChange = {
+                onAnswersChange(listOf(AnswerInput.Content(index = it.first, content = it.second)))
+            })
```

Comment:
Could you please try this? 😄 It's better for readability IMO

```suggestion
            onTextChange = { (index, content) ->
                onAnswersChange(listOf(AnswerInput.Content(index = index, content = content)))
            })
```

Reply:
Good to know this! Fixed in https://github.com/suho/kmm-ic/pull/120/commits/3ac803117d682a8efb02d784f7c71974f3358abd